### PR TITLE
sdk: python: Move examples into sdk’s path

### PR DESCRIPTION
**Pull request by helderco**
Signed-off-by: Helder Correia

### CODE REVIEW

File: sdk/python/examples/client-simple/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Python 3.10+
+- [Docker](https://docs.docker.com/engine/install/)
+
+## How to use?
+
+```sh
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --pre dagger-io
+python say.py "Simple is better than complex"
```
Comment (gerhard) [inline sdk/python/examples/client-simple/README.md:14]: That took a long time to return the first time (`21s`). Second run is 10x quicker (`2s`). Some feedback during those initial `21s` would have helped. Not an issue, just mentioning it.

image

File: sdk/python/examples/client-simple/say_sync.py [hunk 2]
```diff
@@ -24,4 +24,7 @@ def main(args: list[str]):
 
 
 if __name__ == "__main__":
+    if not sys.argv[1:]:
+        print("What do you want to say?", file=sys.stderr)
+        sys.exit(1)
```
Comment (gerhard) [inline sdk/python/examples/client-simple/say_sync.py:29]: 👍

image

**Review by gerhard — APPROVED** [review]
🙏 Merge (hands) when ready!

File: sdk/python/examples/client-simple/README.md (hunk 1 above)
Comment (helderco) [reply sdk/python/examples/client-simple/README.md:14]: say-status

**Pull request merged**